### PR TITLE
feat(index): add willBeCouple var

### PR DESCRIPTION
**Pull request by SaidBySolo**

### CODE REVIEW

**Comment by wonderlandpark** [comment]
Missing typing

File: index.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 const saidbysolo = {
   isSolo: true,
   isCouple: false,
+  willBeCouple: true,
```
Comment (wonderlandpark) [inline index.js:4]:
```suggestion
  willBeCouple: false,
```

**Review by wonderlandpark — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (SaidBySolo) [reply index.js:4]: why false?

Comment (wonderlandpark) [inline index.js:4]: Conflict with https://github.com/wonderlandpark/saidbysolo/blob/3212ce9e4289760e281aacba369883300a062ce1/index.js#L5

**Review by wonderlandpark — CHANGES_REQUESTED** [review]

**Pull request closed without merging**